test/integration/password_reset_test.rb

```diff
@@ -1,19 +1,25 @@
 require 'test_helper'
+include ActiveJob::TestHelper
 
 class PasswordResetTest < SystemTest
   setup do
     @user = create(:user, handle: nil)
+    @old_queue_adapter = queue_adapter
+    ActiveJob::Base.queue_adapter = :test
```

Comment:
Yes. I would still trust rails to pick smart defaults. Can you please explain why do you think we should change it to `:test` everywhere?